packages/syft/src/syft/types/dicttuple.py

```diff
@@ -238,6 +238,14 @@ def __getitem__(self, __key: _KT | slice | SupportsIndex, /) -> _VT | Self:
     def __repr__(self) -> str:
         return f"{self.__class__.__qualname__}{super().__repr__()}"
 
+    def _repr_html_(self) -> str:
+        super_repr_html = getattr(super(), "_repr_html_", None)
+        if super_repr_html is None:
+            return super().__repr__()
```

Comment:
nitpick: should be `self.__repr__()` here, or more pythonic

```suggestion
            return repr(self)
```